website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1029,21 +1374,63 @@ $ curl \
 }
 ```
 
-## Add tags to a workspace
+_List key-value tags bound directly to the workspace:_
```

Comment:
Is there a reason why we italicize these descriptions? i don't remember us doing this for anything else...
<img width="585" alt="image" src="https://github.com/user-attachments/assets/77f15f0b-1a05-4fae-8d58-b9475267e2fd">

Alternatively we could use headings for these

<img width="719" alt="image" src="https://github.com/user-attachments/assets/7b0d477a-f46d-4857-8bc9-4b00230fe44c">


````
<CodeBlockConfig hideClipboard heading="List key-value tags bound directly to the workspace">

```json
{
  "data": [
    {
      "id": "tb-RKs9JSC2lInns32K",
      "type": "tag-bindings",
      "attributes": {
        "key": "ws1-key",
        "value": "ws1-value"
      }
    }
  ]
}
```

</CodeBlockConfig>


<CodeBlockConfig hideClipboard heading="List all key-value tags bound to the workspace, including those inherited from the parent project">

```json
{
  "data": [
    {
      "type": "effective-tag-bindings",
      "attributes": {
        "key": "ws1-key",
        "value": "ws1-value"
      }
    },
    {
      "type": "effective-tag-bindings",
      "attributes": {
        "key": "key1",
        "value": "value1"
      }
    },
    {
      "type": "effective-tag-bindings",
      "attributes": {
        "key": "key2",
        "value": "value2"
      }
    }
  ]
}
```

</CodeBlockConfig>
````